google-cloud-compute-v1/README.md

```diff
@@ -20,7 +20,7 @@ In order to use this library, you first need to go through the following steps:
 
 1. [Select or create a Cloud Platform project.](https://console.cloud.google.com/project)
 1. [Enable billing for your project.](https://cloud.google.com/billing/docs/how-to/modify-project#enable_billing_for_a_project)
-1. {file:AUTHENTICATION.md Set up authentication.}
+2. [Set up authentication.](AUTHENTICATION.md)
```

Comment:
Leave the step number at 1. (Markdown will renumber them when it renders the page.)